src/ghost-upload-theme.ts

```diff
@@ -26,17 +27,39 @@ const config: Config = createConfig(environment);
 async function start() {
   const ghostApi: GhostApi = new GhostApi(config);
 
-  await ghostApi.init();
+  if (args.initGhost) {
+    debugLog(`Initializing Ghost...`);
+    await ghostApi.init();
+    debugLog(`Initialization success.`);
+  }
 
+  debugLog(`Logging in...`);
+  await ghostApi.login();
+  debugLog(`Login success.`);
+
+  debugLog(`Uploading Theme...`);
   const uploadThemeRes = await ghostApi.uploadTheme(() =>
     getStreamForPath(args.themePath)
   );
 
   const [theme] = uploadThemeRes.themes;
 
-  console.log(
-    `Theme "${theme.name}" has been uploaded (v ${theme.package.version})`
-  );
+  debugLog(`Theme "${theme.name}" uploaded (v${theme.package.version}).`);
+  debugLog('Activating theme...');
+  await ghostApi.activateTheme(theme);
```

Comment:
Do we always want to activate the uploaded theme? Having an flag for that too might be :ok_hand: